CairoMakie/src/screen.jl

```diff
@@ -204,6 +204,7 @@ Base.size(screen::Screen) = round.(Int, (screen.surface.width, screen.surface.he
 # we render the scene directly, since we have
 # no screen dependent state like in e.g. opengl
 Base.insert!(screen::Screen, scene::Scene, plot) = nothing
+Base.insert!(screen::Screen, scene::Scene, plot::Plot{plotlist}) = nothing
```

Comment:
```suggestion
# to resolve method ambiguity, since this method is defined for MakieScreen and PlotList.
Base.insert!(screen::Screen, scene::Scene, plot::Plot{plotlist}) = nothing 
```